sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/pkversion/LeaseRenewerImpl.java

```diff
@@ -65,9 +65,9 @@ public Mono<Void> run(CancellationToken cancellationToken) {
             .then()
             .doOnError(throwable -> {
                 if (throwable instanceof LeaseLostException) {
-                    logger.info("Partition {}: renew lease loop failed.", this.lease.getLeaseToken(), throwable);
+                    logger.info("Partition " + this.lease.getLeaseToken() + ": renew lease loop failed.", throwable);
```

Comment:
Does logger info has this overload with exception logging? if not, we can use warn here or just log the exception message as getMessage() with concatenation.